lib/cli/bin/generate.js

```diff
@@ -1,201 +1,49 @@
-import updateNotifier from 'update-notifier';
 import program from 'commander';
 import chalk from 'chalk';
-import detect from '../lib/detect';
-import hasYarn from '../lib/has_yarn';
-import types from '../lib/project_types';
-import { commandLog, codeLog, paddedLog, installDeps } from '../lib/helpers';
 import pkg from '../package.json';
-import angularGenerator from '../generators/ANGULAR';
-import meteorGenerator from '../generators/METEOR';
-import reactGenerator from '../generators/REACT';
-import reactNativeGenerator from '../generators/REACT_NATIVE';
-import reactNativeScriptsGenerator from '../generators/REACT_NATIVE_SCRIPTS';
-import reactScriptsGenerator from '../generators/REACT_SCRIPTS';
-import sfcVueGenerator from '../generators/SFC_VUE';
-import updateOrganisationsGenerator from '../generators/UPDATE_PACKAGE_ORGANIZATIONS';
-import vueGenerator from '../generators/VUE';
-import polymerGenerator from '../generators/POLYMER';
-import webpackReactGenerator from '../generators/WEBPACK_REACT';
-import mithrilGenerator from '../generators/MITHRIL';
-import markoGenerator from '../generators/MARKO';
-import htmlGenerator from '../generators/HTML';
-import riotGenerator from '../generators/RIOT';
+import initiate from '../lib/initiate';
+import yarnSpawnSync from '../lib/yarn_spawn_sync';
+import { codeLog } from '../lib/helpers';
 
 const logger = console;
 
+if (process.argv[1].includes('getstorybook')) {
+  logger.log(chalk.yellow('WARNING: getstorybook is deprecated.'));
+  logger.log(chalk.yellow('To install storybook, use the following command instead:\n'));
+  codeLog(['storybook init']);
+  logger.log();
+  process.exit(0);
```

Comment:
The depreciation should actually work. I assume `getstorybook` should do what `storybook init` does.

Reply:
Sounds logical, will do